src/photos.cs

```diff
@@ -1162,22 +1167,35 @@ interface PHPhotoLibrary {
 		[TV (13,0), Mac (10,15), iOS (13,0)]
 		[Export ("unregisterAvailabilityObserver:")]
 		void Unregister (IPHPhotoLibraryAvailabilityObserver observer);
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("PHLocalIdentifiersErrorKey")]
+		NSString PHLocalIdentifiersErrorKey { get; }
 	}
 
 	[Mac (10,13)]
-	[NoTV][NoiOS]
```

Comment:
If the category has been added to iOS and tvOS now, you'll need the corresponding availability attributes (and that way you won't need them on each member either).